haas/api.py

```diff
@@ -1073,6 +1089,7 @@ def show_node(nodename):
                                      attachment.network.label)
                                     for attachment in n.attachments]),
                   } for n in node.nics],
+        'metadata': {m.label: m.value for m in node.metadata}
```

Comment:
This syntax was introduced in python 2.7; this is going to break on 2.6, which we *might* care about if we want to support CentOS 6. I'll leave it to the rest of the team whether or not we care. I don't think we're really testing with it, so maybe moot. But if we want to be 2.6 compatible you can do:

    dict([(m.label, m.value) for m in node.metadata])